tests/test_api_keys.py

```diff
@@ -0,0 +1,108 @@
+"""Test cases for the Renault client API keys."""
+from typing import AsyncGenerator
+
+import pytest
+from aiohttp import ClientSession
+from aioresponses import aioresponses  # type: ignore
+
+from renault_api.client import RenaultClient
+from renault_api.const import AVAILABLE_LOCALES
+from renault_api.const import CONF_GIGYA_APIKEY
+from renault_api.const import CONF_GIGYA_URL
+from renault_api.const import CONF_KAMEREON_APIKEY
+from renault_api.const import CONF_KAMEREON_URL
+from renault_api.const import LOCALE_BASE_URL
+from renault_api.exceptions import RenaultException
+from renault_api.helpers import create_aiohttp_closed_event
+from renault_api.helpers import get_api_keys
+
+
+@pytest.fixture
+def renault_client(aiohttp_session: ClientSession) -> RenaultClient:
```

Comment:
```suggestion
def renault_client() -> RenaultClient:
```
don't need it for now.